Sources/PackageLoading/TargetSourcesBuilder.swift

```diff
@@ -109,7 +111,7 @@ public struct TargetSourcesBuilder {
         
         // Check if paths that are enumerated in targets: [] exist
         guard self.fs.exists(at) else {
-            return StringError("\(at) could not be found")
+            return StringError("Could not find")
```

Comment:
should this be StringError("File Not Found: \(at)")

Reply:
No that's currently handled at the call site.
```
self.excludedPaths.forEach { exclude in
        if let message = validTargetPath(at: exclude) {
            let warning = "Invalid Exclude: \(message) '\(exclude)'"
            self.diags.emit(warning: warning)
        }
}
```
So the call site already knows the offending path if we get a non nil return.

Reply:
I see. I would change the text to say "File not found" tho.

Reply:
Right on